lib/std/start.zig

```diff
@@ -65,7 +67,7 @@ comptime {
                 if (!@hasDecl(root, "WinMain") and !@hasDecl(root, "WinMainCRTStartup") and
                     !@hasDecl(root, "wWinMain") and !@hasDecl(root, "wWinMainCRTStartup"))
                 {
-                    @export(&WinStartup, .{ .name = "wWinMainCRTStartup" });
+                    @export(&WinStartup, .{ .name = "wmainCRTStartup" });
```

Comment:
I wonder if always exporting the default Zig main as `wmainCRTStartup` might be a problem in some instances. If the user does `zig build-exe foo.zig --subsystem windows`, linking would fail under the MSVC linker behavior laid out in my above comment.

Is it possible to export the same symbol under two names, `wmainCRTStartup` and `wWinMainCRTStartup`, without any negative side effects? This way it would work regardless of which subsystem is specified at the final link.